[Bug]: TypeScript compilation error

### Software Version

v6.6.0

### Language Version

TypeScript 4.9.5

### Operating System

Mac Venture 13.4

### What happened?

After installing the `@easypost/api` package and trying to run my server, I was presented with the following build error after importing it:

```
../node_modules/@easypost/api/types/Pickup/PickupCreateParameters.d.ts:1:24 - error TS7016: Could not find a declaration file for module '../../src/models/pickup_rate'. '<PROJECT>/node_modules/@easypost/api/src/models/pickup_rate.js' implicitly has an 'any' type.

1 import PickupRate from '../../src/models/pickup_rate';
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in ../node_modules/@easypost/api/types/Pickup/PickupCreateParameters.d.ts:1
```

### What was expected?

The package to not cause build issues for my server.

### Sample Code

fix is in #391 

### Relevant logs

_No response_

--- a/types/Pickup/PickupCreateParameters.d.ts
+++ b/types/Pickup/PickupCreateParameters.d.ts
@@ -1,4 +1,4 @@
-import PickupRate from '../../src/models/pickup_rate';
+import { IPickupRate } from './PickupRate';
 import { Address } from '../Address';
 import { Batch } from '../Batch';
 import { CarrierAccount } from '../Carrier';
@@ -12,7 +12,7 @@ interface BasePickupCreateParameters {
   is_account_address?: boolean | null;
   max_datetime: string;
   min_datetime: string;
-  pickup_rates: PickupRate;
+  pickup_rates: IPickupRate;
   reference?: string | null;
   status: string;
 }

